src/main/java/org/scalatest/tools/maven/AbstractScalaTestMojo.java

```diff
@@ -50,15 +50,15 @@
 
     /**
      * Injected by Maven so that it can be included in the run path.  Should not be user facing.
-     * @parameter expression="${project.build.testOutputDirectory}"
+     * @parameter property="project.build.testOutputDirectory"
```

Comment:
I was conflicted about this. I hate the javadoc stuff, but isn't deprecated or anything. In the case were the scalatest guys accept the PR, and I _don't_ become a maintainer, I thought it would be better to leave them be.